.circleci/config.yml

```diff
@@ -384,9 +495,11 @@ jobs:
               export PATH=$HOME/miniconda/bin:/usr/local/cuda/bin:$PATH
               . activate habitat; cd habitat-sim
 
-              export PYTHONPATH=$PYTHONPATH:$(pwd)
+              export PYTHONPATH=$(pwd):$PYTHONPATH
```

Comment:
This is the same as `PYTHONPATH=$PYTHONPATH:$(pwd)` since `$PYTHONPATH` is empty.  We know the build bellow is being picked up since codecov shows coverage the GPU->GPU transfer part of simulator.py

Reply:
Ah, you are right. I forgot that `sys.path` and PYTHONPATH are different!